include/dsVideoPort.h

```diff
@@ -231,7 +232,8 @@ dsError_t  dsIsDisplaySurround(intptr_t handle, bool *surround);
 /**
  * @brief Gets the surround mode of video port
  * 
- * This function is used to get the surround mode of the specified video port.
+ * For source devices, this function is used to get the surround mode of the specified video port.
+ * For sink devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
```

Comment:
Do we need to mention like this?  Actually the dsSURROUNDMode_t  is incomplete. 
Can we check if this is used by middleware or not?

Reply:
dsGetSurroundMode() does not have HAL implementation for sink

Reply:
@ddevadas53276 ,  do you want to double-check this for implementation?

Reply:
For Sink devices no use case & impacts.